crates/tests/tests/integration.rs

```diff
@@ -138,8 +142,35 @@ async fn test_trbfv_actor() -> Result<()> {
         .build()
         .await?;
 
+    let chain_id = 1u64;
+    bus.send(EnclaveEvent::from(ConfigurationUpdated {
+        parameter: "ticketPrice".to_string(),
+        old_value: U256::ZERO,
+        new_value: U256::from(10_000_000u64),
+        chain_id,
+    }))
+    .await?;
+
     for node in nodes.iter() {
-        adder.add(&node.address()).await?;
+        let addr = node.address();
```

Comment:
should we consider using the `setup_score_sortition()` function in the other test?